orbstation/code/mob/species/language/secret_languages.dm

```diff
@@ -0,0 +1,47 @@
+/datum/language
+	var/secret = FALSE //whether the language should be hidden on the language menu
+
+// Aphasia - language only granted by brain damage
+/datum/language/aphasia
+	secret = TRUE
+
+// Beach Bum - language only granted by drugs or a special ghost role
+/datum/language/beachbum
+	secret = TRUE
+
+// Buzzwords - special language for spiders and flypeople
+/datum/language/buzzwords
+	secret = TRUE
+
+// Codespeak - special Syndicate language
+/datum/language/codespeak
+	secret = TRUE
+
+// Drone - you are not a funny little robot
+/datum/language/drone
+	secret = TRUE
+
+// Encoded audio language - you are not a funny big robot either
+/datum/language/machine
+	secret = TRUE
+
+// Mushroom - this language shouldn't even be in the game still
+/datum/language/mushroom
+	secret = TRUE
+
+// Narsian - this one being secret seems obvious
+/datum/language/narsie
+	secret = TRUE
+
+// Piratespeak - it'd be a bit goofy to allow this one
+/datum/language/piratespeak
+	secret = TRUE
```

Comment:
We have at least one character who would want this and it seems generally sort of harmless, but I don't care much either way.

Reply:
Alright, that's fair enough. It could lead to fun moments if pirates attack and someone actually took this.